adam-core/src/main/scala/org/bdgenomics/adam/rdd/feature/FeatureRDD.scala

```diff
@@ -109,6 +114,28 @@ private object FeatureOrdering extends FeatureOrdering[Feature] {}
 object FeatureRDD {
 
   /**
+   * A GenomicRDD that wraps an RDD of Feature data.
+   *
+   * @param rdd An RDD of genomic Features.
+   * @param sequences The reference genome this data is aligned to.
+   */
+  def apply(rdd: RDD[Feature],
+            sequences: SequenceDictionary): FeatureRDD = {
+    new RDDBoundFeatureRDD(rdd, sequences)
```

Comment:
I only have a strong preference for consistency. I think elsewhere we mostly use the `apply` method. These are nits of course, so there's no real obligation to change them.